### PR TITLE
[zh-cn] Create translation of html `maxlength` attribute

### PR DESCRIPTION
**Pull request by MaxtuneLee**
### Description
Create translation of html maxlength attribute

### Motivation
There is no translation for `maxlength` attribute currently

### Related issues and pull requests
Relates to https://github.com/mdn/translated-content/issues/17632

### CODE REVIEW

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+{{Compat}}
+
+## 参见
+
+- [`minlength`](/en-US/docs/Web/HTML/Attributes/minlength)
+- [`size`](/en-US/docs/Web/HTML/Attributes/size)
+- [`pattern`](/en-US/docs/Web/HTML/Attributes/pattern)
+- [约束验证](/zh-CN/docs/Web/HTML/Constraint_validation)
+- [表单数据校验](/zh-CN/docs/Learn/Forms/Form_validation)
+- {{htmlelement('input')}}
```
[the remark this answers is not in the document]
Comment (MaxtuneLee) [reply files/zh-cn/web/html/attributes/maxlength/index.md:45]: Why? nothing changed in the suggestion.

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+{{EmbedInteractiveExample("pages/tabbed/attribute-maxlength.html", "tabbed-shorter")}}
+
+## 示例
+
+```html
+<input type="password" maxlength="4" />
+```
+
+{{EmbedLiveSample('Examples', '100%', 200)}}
```
Comment (T34-active) [inline files/zh-cn/web/html/attributes/maxlength/index.md:28]:
```suggestion
{{EmbedLiveSample('示例', '100%', 200)}}
```

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+slug: Web/HTML/Attributes/maxlength
+l10n:
+  sourceCommit: 2b112aef57df0649462db5d9f47d782a7aa1f25c
+---
+
+{{HTMLSidebar}}
+
+`maxlength` 属性定义了用户可以在 {{htmlelement('input')}} 或 {{htmlelement('textarea')}} 中输入的最大[字符串长度](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length)。该属性的值必须是 0 或更高的整数。
+
+字符串长度是以 UTF-16 编码单元为单位进行测量的，通常（[对于大多数语言脚本](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length#strings_with_length_not_equal_to_the_number_of_characters)）等同于字符数。如果未指定 `maxlength`，或指定了无效值，则输入没有最大长度限制。
```
Comment (T34-active) [inline files/zh-cn/web/html/attributes/maxlength/index.md:12]:
```suggestion
字符串长度是以 UTF-16 编码单元为单位进行测量的，通常（[对于大多数语言脚本](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length#长度不等于字符数的字符串)）等同于字符数。如果未指定 `maxlength`，或指定了无效值，则输入没有最大长度限制。
```

File: files/zh-cn/web/html/attributes/maxlength/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/zh-cn/web/html/attributes/maxlength/index.md:45]:
```suggestion
- {{htmlelement('input')}}

```

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+title: HTML 属性：maxlength
+slug: Web/HTML/Attributes/maxlength
+l10n:
+  sourceCommit: 2b112aef57df0649462db5d9f47d782a7aa1f25c
+---
+
+{{HTMLSidebar}}
+
+`maxlength` 属性定义了用户可以在 {{htmlelement('input')}} 或 {{htmlelement('textarea')}} 中输入的最大[字符串长度](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length)。该属性的值必须是 0 或更高的整数。
```
Comment (PassionPenguin) [inline files/zh-cn/web/html/attributes/maxlength/index.md:10]:
```suggestion
`maxlength` 属性定义了用户可以在 {{htmlelement('input')}} 或 {{htmlelement('textarea')}} 中输入的[字符串长度](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length)的最大值。该属性的值必须是非负整数。
```

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+字符串长度是以 UTF-16 编码单元为单位进行测量的，通常（[对于大多数语言脚本](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length#长度不等于字符数的字符串)）等同于字符数。如果未指定 `maxlength`，或指定了无效值，则输入没有最大长度限制。
```
Comment (PassionPenguin) [inline files/zh-cn/web/html/attributes/maxlength/index.md:12]:
```suggestion
字符串长度是以 UTF-16 编码单元为单位进行计算的，通常（[对于大多数语言脚本](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length#长度不等于字符数的字符串)）等同于字符个数。如果未指定 `maxlength`，或指定了无效值，则输入将没有最大长度限制。
```

File: files/zh-cn/web/html/attributes/maxlength/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/zh-cn/web/html/attributes/maxlength/index.md:45]: image

mdn-linter 不灵敏

**Review by PassionPenguin — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/html/attributes/maxlength/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MaxtuneLee) [reply files/zh-cn/web/html/attributes/maxlength/index.md:45]: > image
> 
> mdn-linter 不灵敏

原来如此，感谢 😃

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+title: HTML 属性：maxlength
+slug: Web/HTML/Attributes/maxlength
+l10n:
+  sourceCommit: 2b112aef57df0649462db5d9f47d782a7aa1f25c
+---
+
+{{HTMLSidebar}}
+
+`maxlength` 属性定义了用户可以在 {{htmlelement('input')}} 或 {{htmlelement('textarea')}} 中输入的[字符串长度](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length)的最大值。该属性的值必须是非负整数。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/maxlength/index.md:10]:
```suggestion
**`maxlength`** 属性定义了用户可以在 {{htmlelement('input')}} 或 {{htmlelement('textarea')}} 中输入的[字符串长度](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length)的最大值。该属性的值必须是非负整数。
```

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+字符串长度是以 UTF-16 编码单元为单位进行计算的，通常（[对于大多数语言脚本](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length#长度不等于字符数的字符串)）等同于字符个数。如果未指定 `maxlength`，或指定了无效值，则输入将没有最大长度限制。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/maxlength/index.md:12]: https://developer.mozilla.org/zh-CN/docs/Glossary/Code_unit

```suggestion
字符串长度是以 UTF-16 码元为单位进行计算的，通常（[对于大多数语言脚本](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/length#长度不等于字符数的字符串)）等同于字符个数。如果未指定 `maxlength`，或指定了无效值，则输入将没有最大长度限制。
```

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+任何 `maxlength` 的值必须大于或等于 [`minlength`](/zh-CN/docs/Web/HTML/Attributes/minlength) 的值（如果存在且有效）。如果字段文本值的长度超过了 `maxlength` 的 UTF-16 编码单元长度，输入将无法通过约束验证。约束验证仅在用户更改值时应用。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/maxlength/index.md:14]:
```suggestion
任何 `maxlength` 的值必须大于或等于 [`minlength`](/zh-CN/docs/Web/HTML/Attributes/minlength) 的值（如果存在且有效）。如果字段文本值的长度超过了 `maxlength` 的 UTF-16 码元长度，输入将无法通过约束验证。约束验证仅在用户更改值时应用。
```

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+### 约束验证
+
+虽然浏览器通常会阻止用户输入超过 `maxlength` 属性允许的文本长度，但如果输入的长度确实超出了 `maxlength` 的限制，{{domxref("ValidityState.tooLong", "tooLong")}} 属性会在 {{domxref("ValidityState")}} 对象中返回 `true`。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/maxlength/index.md:18]:
```suggestion
虽然浏览器通常会阻止用户输入超过 maxlength 属性允许的文本长度，但如果输入的长度确实超出了 maxlength 的限制，{{domxref("ValidityState")}} 对象的 {{domxref("ValidityState.tooLong", "tooLong")}} 属性会返回 true。
```

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+{{Compat}}
+
+## 参见
+
+- [`minlength`](/zh-CN/docs/Web/HTML/Attributes/minlength)
+- [`size`](/zh-CN/docs/Web/HTML/Attributes/size)
+- [`pattern`](/zh-CN/docs/Web/HTML/Attributes/pattern)
+- [约束验证](/zh-CN/docs/Web/HTML/Constraint_validation)
+- [表单数据校验](/zh-CN/docs/Learn/Forms/Form_validation)
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/maxlength/index.md:44]:
```suggestion
- [表单验证](/zh-CN/docs/Learn/Forms/Form_validation)
```

File: files/zh-cn/web/html/attributes/maxlength/index.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
+---
+title: HTML 属性：maxlength
+slug: Web/HTML/Attributes/maxlength
+l10n:
+  sourceCommit: 2b112aef57df0649462db5d9f47d782a7aa1f25c
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/maxlength/index.md:5]: 当前文档的最新的 commit sha：

- https://github.com/mdn/content/commit/067a40e4ed27ea6e1f3b8bbfec15cd9dc3078f4c

```suggestion
  sourceCommit: 067a40e4ed27ea6e1f3b8bbfec15cd9dc3078f4c
```

**Review by yin1999 — COMMENTED** [review]
建议：

**Review by yin1999 — APPROVED** [review]
Thanks @MaxtuneLee

**Pull request merged**